asapdiscovery-alchemy/asapdiscovery/alchemy/cli/alchemy.py

```diff
@@ -936,6 +953,10 @@ def predict(
     )
     console.print(message)
 
+    # if requested, write an SDF of the top n compounds' docked poses
+    if write_top_n_poses > 0:
+        _ = get_top_n_poses(absolute_df, ligands, write_top_n_poses, console)
```

Comment:
```suggestion
        _ = get_top_n_poses(absolute_df, ligands, write_top_n_poses, console, write_file=True)
```

Make the file writing side effect clearer.